chef-helpers/lib/chef-helpers/service.rb

```diff
@@ -0,0 +1,63 @@
+#
+# Copyright:: Copyright 2018-2018, Chef Software Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+module ChefHelpers
+  # NOTE: these are mixed into the service resource+providers specifically and deliberately not
+  # injected into the global namespace
+  module Service
+    extend self
```

Comment:
FIXME: need to add Train shit back here since this was written in Chef 14 before train